pkg/visor/rpc.go

```diff
@@ -42,16 +42,18 @@ type RPC struct {
 	log   logrus.FieldLogger
 }
 
-func newRPCServer(v *Visor, remoteName string) *rpc.Server {
+func newRPCServer(v *Visor, remoteName string) (*rpc.Server, error) {
 	rpcS := rpc.NewServer()
 	rpcG := &RPC{
 		visor: v,
 		log:   v.Logger.PackageLogger("visor_rpc:" + remoteName),
 	}
+
 	if err := rpcS.RegisterName(RPCPrefix, rpcG); err != nil {
-		panic(fmt.Errorf("failed to create visor RPC server: %v", err))
+		return nil, fmt.Errorf("failed to create visor RPC server: %v", err)
```

Comment:
I feel this should panic, as it signals a problem with the `rpcG` implementation.

Reply:
Personally, I think error will be more suitable in such cases, cause it gets wrapped, and we're going `log.Fatal` in the outer code anyway. This way error will be easier to understand in logs

Reply:
But panic will direct us to exactly which line it panicked.

Reply:
Let's leave panic here. I don't think that an error will give us big benefits.